dev-docs/howtoguides/release_a_new_version.md

```diff
@@ -39,174 +49,212 @@ If this is your first time releasing ubuntu-advantage-tools, you'll need to do t
-      * *WARNING* Build the package in a clean environment. The reason is that the package
-        will contain everything that is present in the folder. If you are storing credentials or
-        other sensible development information in your folder, they will be uploaded too when we send
-        the package to the ppa. A clean environment is the safest way to perform this.
-
-    b. Edit the changelog
-      * List yourself as the author of this release.
-      * Edit the version number to look like: `27.2~rc1` (`<version>~rc<release-candidate-number>`)
-      * Edit the Ubuntu release name. Start with the ubuntu/devel release.
-      * `git add debian/changelog && git commit -m "throwaway"` - Do **not** push this commit!
-
-    c. `build-package`
-      * This script will generate all the package artefacts in the parent directory as `../out`.
-
-    d. `sbuild-it ../out/<package_name>.dsc`
-      * If this succeeds move on. If this fails, debug and fix before continuing.
-
-    e. Repeat 3.b through 3.d for all supported Ubuntu Releases
-      * The version for series other than devel should be in the form `<version>~<ubuntu-release-number>~rc<release-candidate-number>`
-        This means you must add the release number in the changelog. For example, suppose
-        the devel version is `1.1~rc1`. If you want to build for jammy now, change it to
-        `1.1~22.04~rc1`.
-
-    f. For each release, dput to the staging PPA:
-      * `dput ppa:ua-client/staging ../out/<package_name>_source.changes`
-      * After each `dput` wait for the "Accepted" email from Launchpad before moving on.
-
-## II. Release to Ubuntu (devel and SRU)
-
-> **Note**
-> `kinetic` is used throughout as a reference to the current devel release. This will change.
+### Git branch release strategy overview
+
+This diagram presents the branches that happen during the process of releasing version 42.
+
+<!--
+NOTE: There are fake commits ("Review" and "Release PR") in the mermaid definition to get it to display the review branch in the diagram, but they don't actually get created in a real release.
+-->
+```mermaid
+%%{init: { 'logLevel': 'debug', 'theme': 'base', 'gitGraph': {'showBranches': true, 'showCommitLabel':true,'mainBranchOrder': 2}} }%%
+gitGraph LR:
+  commit id: "v41-feature1"
+  commit id: "v41-feature2" tag: "41"
+  branch review-v42 order: 1
+  commit id: "Review" type: HIGHLIGHT
+  checkout main
+  commit id: "v42-feature1"
+  commit id: "v42-feature2" tag: "42-rc"
+
+  branch next-v43 order: 3
+  commit id: "v43-feature1"
+  commit id: "v43-feature2"
+  checkout main
+  commit id: "v42-fixup1"
+  checkout next-v43
+  commit id: "v43-feature3"
+  checkout main
+  commit id: "v42-fixup2" tag: "42"
+  checkout review-v42
+  merge main id: "Release PR" type:HIGHLIGHT
+  checkout main
+  merge next-v43
+```
+
+`main` is the only long-lived branch.
+
+When we reach a point in `main` where we are ready to release version 42, we do the following:
```

Comment:
I think we should highlight here when we should make the changelog change, just to make it simpler to visualized the whole process

Reply:
Should we also mention here what is the process we need for the other releases we support ? Similar to what you mentioned on the hotfix section ?

Reply:
Yes I'll reference that here.

Reply:
@lucasmoura added!